### DISCUSSION TITLE
could you please help us how can we set these via command prompt.

### DISCUSSION THREAD

**Discussion by RAMPRASADPANDALA** [cefsharp/CefSharp#4977] [Q&A]
As part of https://bitbucket.org/chromiumembedded/cef/commits/ff33d7f72144 we do see below properties were removed .  We have to set them via programmatically

Our existing code is like below.

browserSettings.FileAccessFromFileUrls = CefState.Disabled;
                browserSettings.UniversalAccessFromFileUrls = CefState.Disabled;
                browserSettings.WebSecurity = CefState.Enabled;
browserSettings.ApplicationCache = CefState.Enabled;

Could you please help us understand how we can achieve the same functionality if we don't set what will be considered the default values for these properties?

**Comment by amaitland** [comment]
See https://github.com/cefsharp/CefSharp/wiki/General-Usage#cefsettings-and-browsersettings for examples of using `CefCommandLineArgs` to set args
